TreeDB/internal/rootpublication/resource_append_mutation_test.go

```
@@ -468,6 +482,545 @@ func TestMergeAppendOnlyLogicalObligationsKeepsSmallBuilderLinear(t *testing.T)
+		t.Fatal(err)
+	}
+	defer merged.Release()
+	if merged.Len() != stableResourceEntryLinearLookupLimit+2 || len(merged.Tokens()) != merged.Len() || len(merged.PhysicalDescriptors()) != merged.Len() {
+		t.Fatalf("mixed canonical sizes len=%d tokens=%d physical=%d", merged.Len(), len(merged.Tokens()), len(merged.PhysicalDescriptors()))
+	}
+	if err := merged.validateResolved(); err != nil {
+		t.Fatal(err)
+	}
+	var current, admitted *StableResourceDescriptor
+	for _, descriptor := range merged.Descriptors() {
+		descriptor := descriptor
```

golangci-lint (2.12.2)

[error] The copy of the 'for' variable "descriptor" can be deleted (Go 1.22+)

(copyloopvar)